core/logic/src/main/java/org/apache/syncope/core/logic/ReportTemplateLogic.java

```diff
@@ -84,6 +85,12 @@ public ReportTemplateTO transform(final ReportTemplate input) {
 
     @PreAuthorize("hasRole('" + StandardEntitlement.REPORT_TEMPLATE_CREATE + "')")
     public ReportTemplateTO create(final String key) {
+        for (ReportTemplateTO reportTemplateTO: list()) {
```

Comment:
Take a look at [ResourceLogic](https://github.com/apache/syncope/blob/master/core/logic/src/main/java/org/apache/syncope/core/logic/ResourceLogic.java#L121-L123) and do accodingly.